src/main/java/io/jenkins/plugins/analysis/core/restapi/BlameApi.java

```diff
@@ -0,0 +1,55 @@
+package io.jenkins.plugins.analysis.core.restapi;
+
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+import io.jenkins.plugins.forensics.blame.FileBlame;
+
+/**
+ * Remote API for a {@link FileBlame}. Simple Java Bean that exposes several methods of an {@link FileBlame} instance.
+ *
+ * @author Kezhi Xiong
+ */
+@ExportedBean
+public class BlameApi {
+    private FileBlame fileBlame;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'fileBlame' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5045840)